plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetFieldSetterFactory.java

```diff
@@ -51,6 +51,15 @@ public FieldSetter create(SettableStructObjectInspector rowInspector, Object row
         return super.create(rowInspector, row, field, type);
     }
 
+    @Override
+    protected Object getField(Type type, Block block, int position)
```

Comment:
> Fix Parquet timestamps by overriding getField

- remove "by overriding getField" from commit title (it's redundant in title, and just update the message)
- add something about nested fields to the title -- since Parquet timestamps were generally good, unless nested

Reply:
Changed to `Fix timestamps in arrays/maps/structs for Parquet`.